api-reference/_hidden/DataChange/index.md

```diff
@@ -4,7 +4,12 @@ type: Number
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+[tags] ctp
+
+Zero-based index of a new row.
```

Comment:
```suggestion
Zero-based index of a new row on the [current page]({currentpath}/#pageIndex). Applies only if the [type]({currentpath}/#type) is *"insert"*.
```

Reply:
> on the [current page]

@RomanTsukanov what do you mean by 'current page' in the case with virtual scrolling? Viewport or batch of loaded data rows or something else?

Reply:
You tell me :) I consider the page specified by pageIndex to be the current page.